src/main/java/com/github/manolo8/darkbot/core/manager/StatsManager.java

```diff
@@ -123,6 +128,14 @@ private void registerImpl(Key key, StatImpl stat) {
         statistics.put(StatKey.of(key), stat);
     }
 
+    public static Collection<? extends StatsAPI.Key> getStatKeys() {
+        return Collections.unmodifiableSet(getInstance().statistics.keySet());
+    }
+
+    public static StatsManager getInstance() {
```

Comment:
preferrably get this thru a reference to `Main.INSTANCE` instead of creating more static stuff

Reply:
Done